Userland/Libraries/LibVideo/VP9/Parser.cpp

```diff
@@ -1399,4 +1410,8 @@ void Parser::dump_info()
     outln("Interpolation filter: {}", (u8)m_interpolation_filter);
 }
 
+#undef RESERVED_ZERO
+#undef FUNC_ERROR
+#undef WRAP_ERROR
```

Comment:
What's up with these undefs?

Reply:
My bad, those are leftover from before. (Also, I was apparently writing the file thinking of it as a header, so that was pointless anyway.)